web/packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -832,3 +870,17 @@ function rewrapResolveError(
     );
   };
 }
+
+const APP_UPDATER_STATE_KEY = 'appUpdater';
```

Comment:
nit: define the auto update related constants in a single place. Maybe in the appUpdater along with `const TELEPORT_TOOLS_VERSION_ENV_VAR = 'TELEPORT_TOOLS_VERSION';` ?

Reply:
I’m not certain if these constants are closely related to each other. `TELEPORT_TOOLS_VERSION_ENV_VAR` is required to determine the version, while `APP_UPDATER_STATE_KEY` defines the location within the state where the `AppUpdater` class can read and write. However, the updater doesn’t need to be aware of this key; it only requires an interface to interact with that state. For that reason, I believe it's reasonable to define `APP_UPDATER_STATE_KEY` externally.